### DISCUSSION TITLE
百度网盘第三方登录刷新令牌失败

### DISCUSSION THREAD

**Discussion by Quackgrass** [AlistGo/alist#1270] [General]
根据文档[添加账号->百度网盘](https://alist-doc.nn.ci/docs/driver/baidu)获取刷新令牌，获取QQ登录的百度网盘失败。此前使用邮箱登录的获取成功。

错误提示为 

> 错误代码：not_login
> 错误信息：User does not login
> 详细描述：当前用户未登录

![image_263](https://user-images.githubusercontent.com/42032091/175781462-77e57ba8-8251-4d2a-a64c-a60acb987bce.png)

**Comment by anwen-anyi** [comment]
QAQ 你换手机号登录呗? 提前登录好再获取~

**Reply by Quackgrass** [reply]
> 此前使用邮箱登录的获取成功。

并不是操作失误。我是先登录后获取令牌的顺序操作的，用邮箱登录的账号获取令牌成功了，再使用 QQ 登录的账号获取时失败了。

因为提示的“当前用户未登录”信息过于模糊且与实际不符，故提问求助。

**Comment by supermaruko** [comment]
我用邮箱登录 也是同样问题 好奇怪

**Reply by supermaruko** [reply]
(⊙﹏⊙) 申请了那个百度什么网盘开发api 然后删了百度的cookie重新登陆 就好了 大家可以试下
